pages/category/index.vue

```diff
@@ -701,18 +854,169 @@ export default {
   &__header {
     font-size: var(--font-size--sm);
     font-weight: bold;
+    @include for-mobile {
```

Comment:
default design should be for mobile-only, hence remove @include for-mobile

Reply:
Resolved